utils/coreutils/tableutils.go

```diff
@@ -216,6 +231,38 @@ func PrepareTable(rows interface{}, emptyTableMessage string, printExtended bool
 	return tableWriter, nil
 }
 
+func PrintTableWithBorderless(rows interface{}, title string, footer string, emptyTableMessage string, printExtended bool) (err error) {
```

Comment:
aren't we using this in jfrog-cli-artifactory? since we already have tableutils why do we need pterm dep in jfrog-cli-artifactory?